### PR TITLE
Feature/mx 1646 synopse update

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context

- [x] issue to resolve: some resource_extended-data-use don't have variables(metadata review pending)
-> MX-1675 created for metadata review

-refactoring of the extractor is out of scope for this ticket, suggestions can go into [MX-1667]

Changes:
-switched from hard coded default values to mapping extractor
-adapt extractor to old mapping
-adapt extractor to new mapping
-adapt extractor to additional mapping changes

# Added

# Changes
- model-update: synopse

# Deprecated

# Removed

# Fixed

# Security

### CODE REVIEW

File: mex/extractors/synopse/transform.py [hunk 1]
```diff
@@ -615,27 +678,70 @@ def transform_synopse_project_to_activity(
...
         for email in synopse_project.get_contacts()
         if email in contact_merged_ids_by_emails
     ]
+    external_associate = []
+    if synopse_project.externe_partner:
+        for org in synopse_project.externe_partner.split(", "):
+            if org in synopse_organization_ids_by_query_string.keys():
+                external_associate.append(synopse_organization_ids_by_query_string[org])
+            else:
+                extracted_organization = ExtractedOrganization(
```
Comment (mr-kamran-ali) [inline mex/extractors/synopse/transform.py:693]: this is getting repetitive in multiple extractors. We have mx-1532 for this implementation, so we can back this into wikidata convenience function or wikidata module in extractors. for now its fine like this.

**Review by mr-kamran-ali — COMMENTED** [review]
nothing major to complain about, few minor things.

File: mex/extractors/synopse/filter.py [hunk 2]
```diff
@@ -2,32 +2,31 @@
...
+        Generator for filtered synopse studies
     """
-    variables_in_datensatzuebersicht = {s.synopse_id for s in synopse_overviews}
-    for variable in synopse_variables:
-        if variable.synopse_id in variables_in_datensatzuebersicht:
-            yield variable
+    for study in synopse_studies:
+        if study.plattform_adresse not in [
+            "interne Datennutzung",
+            "noch nicht erstellt",
```
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/extractors/synopse/filter.py:24]: I'm not sure whether I can follow. These are just the values in the `access-platform_entity-filter.yaml`. I wouldn't overcomplicate things...

File: mex/extractors/synopse/transform.py (hunk 1 above)
Comment (erichesse) [reply mex/extractors/synopse/transform.py:693]: Yes, I saw you already made a comment under the ticket. We should definitely talk about this at some point

File: mex/extractors/synopse/filter.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mr-kamran-ali) [reply mex/extractors/synopse/filter.py:24]: alright. I just feel like extracting this dynamically from `access-platform_entity-filter.yaml` would be safer and neater.

File: mex/extractors/synopse/transform.py [hunk 3]
```diff
@@ -250,13 +264,13 @@ def transform_synopse_variables_to_mex_variable_groups(
...
 
         label = Text(value=re.sub(r"\s\(\d+\)", "", thema), language=TextLanguage("de"))
-
-        yield ExtractedVariableGroup(
-            containedBy=contained_by,
-            hadPrimarySource=extracted_primary_source.stableTargetId,
-            identifierInPrimarySource=thema,
-            label=label,
-        )
+        if len(contained_by) > 0:
```
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/extractors/synopse/transform.py]: https://github.com/robert-koch-institut/mex-extractors/pull/154/commits/bb7ab0707f0a0761129199360310d822ab5a41c2

File: mex/extractors/synopse/filter.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/extractors/synopse/filter.py:24]: Ok I tried that, but this entity filter object doesn't get validated by the mapping extractor. I think we would need to adapt the mapping extractor first, but I think this is out of scope for this ticket...

[the remark this answers is not in the document]
Comment (mr-kamran-ali) [reply mex/extractors/synopse/filter.py:24]: I see. yeah thats not the task at hand. thanks for trying.

**Review by mr-kamran-ali — APPROVED** [review]
🚀

**Pull request merged**